extensions/ql-vscode/src/queryserver-client.ts

```diff
@@ -60,6 +61,16 @@ export class QueryServerClient extends DisposableObject {
   nextCallback: number;
   nextProgress: number;
   withProgressReporting: WithProgressReporting;
+
+  private readonly _onDidStartQueryServer = [] as ProgressTask<void>[];
```

Comment:
Maybe call this something like `queryServerStartListeners`

Reply:
I was being consistent with our other event fields. But, I'm not fond of the naming convention.